src/main/java/chess/controller/spring/GameController.java

```diff
@@ -0,0 +1,102 @@
+package chess.controller.spring;
+
+import chess.domain.ChessGame;
+import chess.domain.Result;
+import chess.domain.piece.Piece;
+import chess.domain.square.Rank;
+import chess.dto.CreateGameRequestDto;
+import chess.dto.MoveRequestDto;
+import chess.dto.RankDto;
+import chess.service.GameService;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.servlet.ModelAndView;
+
+@Controller
+@RequestMapping("/games")
+public class GameController {
+
+    private final GameService gameService;
+
+    public GameController(final GameService gameService) {
+        this.gameService = gameService;
+    }
+
+    @GetMapping("/{gameId}/play")
+    public ModelAndView renderPlayGame(@PathVariable("gameId") final Long gameId) {
+        final ChessGame chessGame = gameService.findByGameId(gameId);
+        if (chessGame.isEnd()) {
+            return new ModelAndView("redirect:" + String.format("/games/%d/result", gameId));
+        }
+
+        final Map<String, Object> model = new HashMap<>();
+        model.put("turn", chessGame.getTurn());
+        model.put("ranks", makeRanksDto(chessGame));
+        model.put("gameId", gameId);
+        return new ModelAndView("play", model);
+    }
+
+    @PutMapping("/{gameId}/move")
+    public ResponseEntity<Long> movePiece(@PathVariable final Long gameId,
+                                          @RequestBody final MoveRequestDto moveRequestDto) {
```

Comment:
```suggestion
    public ResponseEntity<Long> movePiece(
            @PathVariable final Long gameId, 
            @RequestBody final MoveRequestDto moveRequestDto
    ) {
```

이런식으로 개행을 할 수도 있답니다 😀
개행은 취향이니 반영할 필요는 없어요!

Reply:
와... 이거 진짜 ') {'를 기존처럼 옆에다 써야할지 한칸 밑에 써야할지 고민했었는데 이렇게 비교하니 확실히 아래가 더 구분이 잘되네요!! 감삼당!!